sql/core/src/test/scala/org/apache/spark/sql/DataFrameAggregateSuite.scala

```diff
@@ -1989,13 +1989,15 @@ class DataFrameAggregateSuite extends QueryTest
   }
 
   test("SPARK-43876: Enable fast hashmap for distinct queries") {
-    val df = testData2.distinct()
-    checkAnswer(df, testData2)
-    val output = new java.io.ByteArrayOutputStream()
-    Console.withOut(output) {
-      df.explain("codegen")
+    withSQLConf(SQLConf.ENABLE_TWOLEVEL_AGG_MAP.key -> "true") {
```

Comment:
I think this is enabled by default?

Reply:
This UT will fail in SingleLevelAggregateHashMapSuite, which will disable ENABLE_TWOLEVEL_AGG_MAP. 

https://github.com/apache/spark/blob/master/sql/core/src/test/scala/org/apache/spark/sql/AggregateHashMapSuite.scala#L25-L38